Content.Shared/Physics/CollisionGroup.cs

```diff
@@ -61,16 +61,16 @@ public enum CollisionGroup
     TabletopMachineLayer = Opaque | HighImpassable | BulletImpassable,
 
     // Airlocks, windoors, firelocks
-    GlassAirlockLayer = HighImpassable | MidImpassable | BulletImpassable | InteractImpassable,
+    GlassAirlockLayer = Impassable | BulletImpassable | InteractImpassable,
     AirlockLayer = Opaque | GlassAirlockLayer,
 
     // Airlock assembly
     HumanoidBlockLayer = HighImpassable | MidImpassable,
 
     // Soap, spills
     SlipLayer = MidImpassable | LowImpassable,
-    ItemMask = Impassable | HighImpassable,
-    ThrownItem = Impassable | HighImpassable | BulletImpassable,
+    ItemMask = Impassable,
+    ThrownItem = Impassable | BulletImpassable,
```

Comment:
This makes items able to be thrown through airlocks. I'm not sure if making items able to go on top of machines is wanted, but if it is then it'd probably be best to remove HighImpassable from TabletopMachineLayer.